Wikipedia/Code/ProfileCoordinator.swift

```diff
@@ -175,9 +175,39 @@ final class ProfileCoordinator: NSObject, Coordinator, ProfileCoordinatorDelegat
     }
     
     private func showYearInReview() {
+        if let viewModel, !viewModel.isUserLoggedIn() {
```

Comment:
There's a requirement here:

```
Display log-in / create an account prompt on top of profile menu
```

From my testing, you are first dismissing profile, then displaying the login prompt. Can you fix so that the login prompt displays on top of profile?